README.md

```diff
@@ -27,30 +27,30 @@ The positivity rate for COVID is in high 20%. If you have black stool, loose mot
+मेरी माँ, मामाजी और मौसाजी की COVID से पिछले हफ़्ते में मृत्यु हो गयी। मेरा एक दोस्त ICU में है और उसका ऑक्सीजन `84` है ।  हमसे जो भूल-चूक हुई उसके लिए मैंने ये सूची बनाई है। कृपया ध्यान से पढ़ें। ये जानकारी CDC के निर्देश से अलग है। मैं ये जानता हूँ कि **लोग निर्देशों का पालन तब तक नहीं कर सकते जब तक उन्हें उनके पीछे का अर्थ ना समझाया जाए, इसलिए इस लेख को पूरा पढ़ें**। सबसे नया संस्करण आप यहाँ देख सकते हैं  https://github.com/pushpendre/covid-manual/ । अगर आपको इसमें कोई गलती लगती है तो मुझे बताएँ। **`इस जानकारी को और लोगों के साथ शेयर करें`** । 
 
 
-1. **लगातार भाप ले पर इतनी ज़्यादा नहीं की आपकी साँस की नली को नुक़सान पहुँचे**
+1. **लगातार भाप लें पर इतनी ज़्यादा नहीं कि आपकी श्वास नली को नुक़सान पहुँचे**
 
-वाइरस का असर सबसे पहले हमारी नाक की नली में होता है । नाक में वाइरस बड़ने के बाद वो फेफ़ेड़े में फैलता है । ऊँचा तापमान वाइरस को बडने से रोकता है । ऐसे सोचे जैसे आप अपनी नाक की कीटाणु  रहित कर रहे है । ऐसे भी सोच सकते है की इससे नाक में रउकावट कम होगी ।
+वायरस का असर सबसे पहले हमारी नाक की नली में होता है। यह वायरस नाक में बढ़ने के बाद फेफड़ों में फैलता है। ऊँचा तापमान वायरस को बढ़ने से रोकता है। साथ ही ये नाक की जलन कम करने और जकड़न कम करने में भी सहायक होता है।
 
-2. **वैक्सीन एक इंजेक्शन से बड़कर है**
+2. **वैक्सीन एक इंजेक्शन से बढ़कर है**
 
-वैक्सीन के इंजेक्शन को काम करने के लिए सही हालात (परिस्थितियाँ) चाहिए । वैक्सीन को सही तापमान पर रखना ज़रूरी है । वैक्सीन की सिरिंज साफ़ होनी चाहिए और वैक्सीन लगाने वाला स्टाफ़ स्वस्थ होना चाहिए । वैक्सीन केंद्र पर बाक़ी लोगों से 6 फ़ीट का फ़ासला बना कर रखना बहुत ज़रूरी है । ये आपका हक़ है । वैक्सीन के दूसरे टीके लगने के दो हफ़्ते बाद ही आपका शरीर वाइरस को मारने में सक्षम होता है । उससे पहले आप असुरक्षित है । इसका मतलब वैक्सीन के दूसरे टीके लगने के दो हफ़्ते पहले तक आप को मास्क लगाना ज़रूरी है । और लोगों से ना मिले । अगर आप ये निर्देश नहीं मानते तो आप वाइरसग्रस्त हो सकते है । 
+वैक्सीन के इंजेक्शन को असर करने के लिए सही परिस्थितियाँ आवश्यक हैं। वैक्सीन को सही तापमान पर रखा जाना ज़रूरी है। वैक्सीन की सिरिंज साफ़ होनी चाहिए और वैक्सीन लगाने वाला स्टाफ़ स्वस्थ होना चाहिए। वैक्सीन केंद्र पर बाक़ी लोगों से 6 फ़ीट की दूरी बना कर रखना बहुत ज़रूरी है और आपका कर्तव्य भी है। वैक्सीन के दूसरे टीके लगने के दो हफ़्ते बाद ही आपका शरीर वायरस से लड़ने में सक्षम हो पाता है, उससे पहले आप असुरक्षित हैं। इसका मतलब वैक्सीन का दूसरा टीका लगने के दो हफ़्ते पहले तक आप को मास्क लगाना और दूसरे लोगों से मेल-मिलाप से बचना ज़रूरी है। अगर आप ये निर्देश नहीं मानते तो वैक्सीन का असर लगभग शून्य भी हो सकता है। 
```

Comment:
Same with चाहिए vs आवश्यक 

Instead of  लगभग शून्य let's just go with शून्य । We are already hedging by saying हो सकता है । Double-hedge की ज़रूरत नहीं